_posts/2018/02/2018-02-16-library-carpentry-governance.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+authors: ["Belinda Weaver"]
+title: "Library Carpentry Governance - An Update"
+date: 2018-02-16
+time: "00:00:00"
+category: [ "Library Carpentry", "Governance", "Community Building"]
+---
+
+Eight of us from Australia, the UK, the US, Canada and South Africa met this week via zoom to discuss taking 
+the [Library Carpentry](https://github.com/LibraryCarpentry) project forward.
+
+We discussed the creation of a new website to replace [this one](http://librarycarpentry.github.io/), talked about 
+what constitutes a Library Carpentry workshop, discussed who can teach the material, and what kind of governance structure 
+we need to take the work forward. There was an update on the status of the [IMLS-funded Library Carpentry Coordinator role to be based at CDL](http://www.datacarpentry.org/jobs/).
+
+### The website
+
+[Richard Vankoningsveld](https://twitter.com/richyvk) has done a great draft of 
+a [new website](https://www.richyvk.me/library-carpentry-redesign/). 
+Can people please have a look at it and raise issues that need fixing on the repo itself? 
+We will move it across once we have the material up to date. If you are interested in working on the 
+repo or having rights to edit, ask there by raising an issue.
+
+### What is a workshop?
+
+We need to make a decision about what the 'core' curriculum is for a workshop to be badged Library Carpentry. 
+We have a statement to that effect on the old website. See [What is a workshop?](http://librarycarpentry.github.io/about/) 
+We will revisit that in the coming weeks. Belinda suggested that the core curriculum could be 
+[Intro to Data](http://data-lessons.github.io/library-data-intro/) (which includes regular expressions), [Shell](http://data-lessons.github.io/library-shell/) and [OpenRefine](http://data-lessons.github.io/library-openrefine/). That can be taught within a (fairly intense) 
+day. People can then add extras from the [other modules under development](https://github.com/data-lessons) if they want to have 
+a two-day format, or 
+they can stretch the core curriculum to be taught over two days, which would give more time for people to practise 
+and embed what they have learned.
+
+### One day or two days?
```

Comment:
I'd moved this before the discussion of content, since otherwise people may wonder about the mention of "one day" in the content discussion.